src/main/java/uk/gov/hmcts/reform/cpo/config/HibernateEventListenerRegistryConfiguration.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.hmcts.reform.cpo.config;
+
+import org.hibernate.envers.boot.internal.EnversService;
+import org.hibernate.event.service.spi.EventListenerRegistry;
+import org.hibernate.internal.SessionFactoryImpl;
+import org.hibernate.service.spi.ServiceRegistryImplementor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Configuration;
+
+import javax.annotation.PostConstruct;
+import javax.persistence.EntityManagerFactory;
+
+import static org.hibernate.event.spi.EventType.POST_DELETE;
+
+@Configuration
+public class HibernateEventListenerRegistryConfiguration {
+
+    @Autowired
+    EntityManagerFactory entityManagerFactory;
+
```

Comment:
a small comment here to explain the purpose of the customisation for our functional requirement.

Reply:
+1

Reply:
Fixed in https://github.com/hmcts/cpo-case-payment-orders-api/pull/63/commits/08f287c51a39f85e5dc281a0d29ea6f351a6c432